test/cluster-py/multi.test.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import os
+from threading import Thread
+import time
+import yaml
+from lib.tarantool_server import TarantoolServer
+sys.path.append('../tarantool')
+from mesh_connection import MeshConnection
+from tarantool.const import (
+    SOCKET_TIMEOUT,
+    RECONNECT_DELAY,
+)
+from tarantool.error import NetworkError
+from tarantool.utils import ENCODING_DEFAULT
+
+REPLICA_N = 2
```

Comment:
Nit: It is not a replica, but just separate tarantool instance. Maybe naming should be fixed for other files too.